crates/js-component-bindgen/src/transpile_bindgen.rs

```diff
@@ -511,16 +552,45 @@ impl<'a> Instantiator<'a, '_> {
                 },
             }
         }
+
         self.exports_resource_types = self.imports_resource_types.clone();
-        for (key, item) in &self.resolve.worlds[self.world].exports {
+
+        // Iterate & generate exports
```

Comment:
Same again here, both this and the above are not the right place to insert the stability gating.

Reply:
Moved -- thanks!